lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board
+
+    attr_accessor :cells
+
+    def initialize
+      @cells = Array.new(9, " ")
+    end
+
+    def reset!
+      @cells.clear
```

Comment:
the code on line 11 already replaces the current @cells array so this line is redudant.